front-end/packages/robustar/src/views/ImageList.vue

```diff
@@ -325,6 +348,16 @@ export default {
         { text: 'Incorrectly Classified', value: this.$route.params.split + '_incorrect' },
       ];
     },
+    accuarcy() {
+      const allImageLength = this.testImageList[0];
+      const correctImageLength = this.testImageList[1];
+      const incorrectImageLength = this.testImageList[2];
+
```

Comment:
We can use array destructuring here for simplicity: `const [a, b, c] = this.testImageList`